sonar-scanner.properties

```diff
@@ -0,0 +1,4 @@
+sonar.host.url=http://10.134.116.99:9000
```

Comment:
this is a private IP address for our server... is there a way to not hard-code this in the public image?